pytalibstream/tests/test_atr_keeper.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from pytalibstream.atr_keeper import AtrKeeper
+from pytalibstream.types.hlc import HLC
+
+
+class TestAtrKeeper(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.atr_keeper = AtrKeeper(5)
+        self.sample = range(1, 21)
+
+    def test_add(self):
+        for i in self.sample:
+            candle = HLC.from_list([i + 1, i, i * .90])
+            self.atr_keeper.add(candle)
```

Comment:
we can add some basic check in this test.
I want to setup some talib function so we can compare talib lib results